CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 4.9.0 / TBA
+
+### ✨ New Features
+
+- New Logbook content: A logbook is a type of content that features logbook entries. It displays these entries chronologically as a timeline, with an emphasis on the time and the order.
+- Added real delete of content for workspace manager
+- Added transformation from news into thread
+- The state of the sidebar is now saved in the browser
```

Comment:
- The cache of the browser used now saves the states of the navigation sidebar. Hidden sub-spaces will stay hidden.